gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/matching/single/cypher/operators/PhysicalOperator.java

```diff
@@ -32,4 +32,16 @@
    */
   DataSet<Embedding> evaluate();
 
+  /**
+   * Set the operator description
+   * @param newName operator description
+   */
+  void setName(String newName);
```

Comment:
As you already point out in the JavaDoc, I would prefer calling this `setDescription()` and also add a note on the doc that this is used for Flink operator naming.